webpack.config.js

```diff
@@ -24,6 +24,15 @@ Encore
 
     .enableBuildNotifications()
     .enableSassLoader()
+
+
+    .copyFiles([
```

Comment:
I completely forgot to ask you, can you explain just a bit why you must do that `copyFiles` ?

Reply:
@pifaace These are CKEditor library files required to display WYSIYG editor.

Reply:
@pifaace Not sure what you ask here.... Is there another / better way to do this?

Reply:
I'm wondering why you have to copy by hand all these files, thats my question :)

Reply:
How would you make this files available in /public ?

Reply:
I dont know, I trust your solution, I just wanted some explanations about this ^^

Reply:
@pifaace On the other side, I trust Symfony docs. ^^ https://symfony.com/doc/current/bundles/FOSCKEditorBundle/installation.html

Reply:
you right !